components/webui/client/public/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "ClpStorageEngine": "clp",
-    "ClpQueryEngine": "native",
+    "ClpQueryEngine": "clp",
```

Comment:
we should also configure this in `server_settings_json_updates` in `components/clp-package-utils/clp_package_utils/scripts/start_clp.py`

@coderabbitai create an issue to track this